apps/i18n/common/en_us.json

```diff
@@ -1867,27 +1867,25 @@
   "myProfessionalLearningCourses": "My Professional Learning Courses",
   "myWorkshops": "My Workshops",
 
-  "accountInformationAccountInformation": "Account information",
-  "accountInformationVerifiedTeacher": "Verified Teacher",
-  "accountInformationDisplayName": "Display Name",
-  "accountInformationUsername": "Username",
-  "accountInformationDisplayNameHintTeacher": "This is what your students see.",
-  "accountInformationEmail": "Email address",
-  "accountInformationEmailHint": "Make sure you can receive emails to this email address.",
-  "accountInformationPassword": "Password",
-  "accountInformationPasswordConfirmation": "Password confirmation",
-  "accountInformationCurrentPassword": "Confirm current password",
-  "accountInformationCurrentPasswordHint": "We need your current password to confirm changes.",
-  "accountInformationUpdateAccountInformation": "Update account information",
-  "accountInformationNoPasswordBecauseSponsored": "See your teacher to manage your password.",
-  "accountInformationGender": "Gender (optional)",
-  "accountInformationState": "State",
-  "accountInformationUpdateFieldParentPermissionRequired": "(You must obtain parental permission before updating this field)",
-  "accountInformationUpdateEmail": "Update email",
-  "accountInformationSelectState": "Select state",
-  "accountInformationReviewErrors": "Review errors above and try again.",
-  "accountInformationUpdateSuccess": "Account information successfully updated.",
-  "accountInformationEmailUpdateSuccess": "Email successfully updated.",
+  "accountInformation_accountInformation": "Account information",
+  "accountInformation_verifiedTeacher": "Verified Teacher",
+  "accountInformation_displayName": "Display Name",
+  "accountInformation_username": "Username",
+  "accountInformation_displayNameHintTeacher": "This is what your students see.",
+  "accountInformation_email": "Email address",
+  "accountInformation_emailHint": "Make sure you can receive emails to this email address.",
+  "accountInformation_currentPassword": "Confirm current password",
+  "accountInformation_currentPasswordHint": "We need your current password to confirm changes.",
+  "accountInformation_updateAccountInformation": "Update account information",
+  "accountInformation_noPasswordBecauseSponsored": "See your teacher to manage your password.",
+  "accountInformation_gender": "Gender (optional)",
```

Comment:
Same suggestion as for the username. We already have a Gender string, however there's value into adding a the `Gender (optional)` string, but I would make it a generic string and call it `genderOptional` and added to here: https://github.com/code-dot-org/code-dot-org/blob/fcc0ce8fbcc11d556b3aee3fe2c7cc2d18b81f63/apps/i18n/common/en_us.json#L1348C2-L1352C50